examples/run_examples.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+# set -e
+set -o nounset
+
+origin=$(pwd)
+examples_dir=${1-$origin}
+declare -i exitcode=0
+echo "Running examples in ${examples_dir}..."
+for example in ${examples_dir}/*.py
+do
+    if [[ "${example}" == *"mpi"* ]]
```

Comment:
```suggestion
    if [[ "$example" == *"mpi"* ]]
```

Reply:
Not sure if this `if` should be a bit more strict here (maybe `*-mpi*`?). It currently matches the word `compile` for example.

Reply:
Yeah, it could more strict for sure.  We could just require that MPI examples are `\<whatever\>-mpi.py`.  Does that sound reasonable?